### PR TITLE
SC-5802 - Fixes for the Edusharing

### PR DESCRIPTION
**Pull request by mwolowyk-cap**
# Description

Fixes for the edusharing mentioned in the ticket.
## Links to Tickets or other pull requests
https://ticketsystem.schul-cloud.org/browse/SC-5812

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by mwolowyk-cap** [comment]
@schul-cloud-bot update screenshots

**Comment by mwolowyk-cap** [comment]
@schul-cloud-bot update screenshots

File: src/components/organisms/LernstoreDetailView.vue [hunk 1]
```diff
@@ -102,14 +98,14 @@
 									:key="index"
 									class="meta-text"
 								>
-									<base-link :href="'/content/?q=' + tag" class="tag-link"
+									<base-link :href="'/content/?q=' + tag" class="tag link"
```
Comment (ABialoskorski) [inline src/components/organisms/LernstoreDetailView.vue:101]: here should stay only link?

Comment (mwolowyk-cap) [reply src/components/organisms/LernstoreDetailView.vue:101]: I need to access it in the unit test

**Pull request merged**